charts/super-agent/values.yaml

```diff
@@ -19,6 +19,15 @@ super-agent-deployment:
   # -- In case you don't want to have the license key in you values, this allows you to point to which secret key is the license key located. Can be configured also with `global.customSecretLicenseKey`
   customSecretLicenseKey: ""
 
+  # -- L1 identity client_id to use. Can be configured also with `global.l1IdentityClientId`
```

Comment:
why supporting also "global"? I think that no other chart have it!

Globals are the ones supported by the common library

Reply:
yes, otherwise it is really confusing, someone could think that such global would be available in all charts

Reply:
There you have 69b6a282d20432dc152b74f82da0d37a6e759c2c :)

Reply:
there is still this comment!

Reply:
and the ones below

Reply:
Thanks! :wasn't-me: 20e5d87d66e392c29c4f5a2caa9594de9d66e435

also remved from README